src/common/services/api/order.service.js

```diff
@@ -359,11 +357,7 @@ class Order {
 
   calculateAmountWithFees (originalAmount) {
     originalAmount = parseFloat(originalAmount)
-    return (originalAmount * 0.0235) +
-      (originalAmount * 0.0235 * 0.0235) +
-      (originalAmount * 0.0235 * 0.0235 * 0.0235) +
-      (originalAmount * 0.0235 * 0.0235 * 0.0235 * 0.0235) +
-      originalAmount
+    return originalAmount / 0.9765
```

Comment:
Can you document where this came from in the code? Maybe a comment with some of the equations from https://github.com/CruGlobal/give-web/pull/846#discussion_r424686632 showing how this was derived and that the `.0235` is the 2.35% card processing fee. I forsee myself and others being confused what this magical number is.